### PR TITLE
logger: cache init_data

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

File: selfdrive/loggerd/logger.cc [hunk 1]
```diff
@@ -253,7 +253,8 @@ int logger_next(LoggerState *s, const char* root_path,
   pthread_mutex_unlock(&s->lock);
 
   // write beggining of log metadata
-  log_init_data(s);
+  auto bytes = s->init_data.asBytes();
```
Comment (adeebshihadeh) [inline selfdrive/loggerd/logger.cc:256]: if we save the bytes, we don't have to do this conversion every time

Comment (deanlee) [reply selfdrive/loggerd/logger.cc]: The kj library has no obvious way to do this,so I used std::vector to make a copy

Comment (deanlee) [reply selfdrive/loggerd/logger.cc]: Maybe the original version will be better.adding a conversion will not have any impact.

Comment (deanlee) [reply selfdrive/loggerd/logger.cc]: There is a function `releaseAsBytes` can be used

Comment (deanlee) [reply selfdrive/loggerd/logger.cc]: sorry,I'm wrong,we can't call releaseAsBytes on kj::Array<capnp::word>:
` error: static_assert failed due to requirement 'sizeof(capnp::word) == sizeof(unsigned char)' "releaseAsBytes() only possible on arrays with byte-size elements (e.g. chars)."`

**Comment by deanlee** [comment]
closed,merged to https://github.com/commaai/openpilot/pull/19942

**Pull request closed without merging**